Update auto-generated documentation

snoopycrimecop opened the pull request:
Repository: openmicroscopy/ome-documentation
Already up-to-date.

Generated by build [OMERO-DEV-latest-docs-autogen#406](https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/406/).

---

--no-rebase

hflynn commented:
Changes all look reasonable, I'll re-run the merge build with this included once Travis passes.

snoopycrimecop committed:
Update auto-generated documentation

Repository: openmicroscopy/ome-documentation
Already up-to-date.

Generated by OMERO-DEV-latest-docs-autogen#407 (https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/407/)
